src/app/components/DoctorCard/DoctorCard.tsx

```diff
@@ -25,6 +28,10 @@ export default function DoctorCard({
   isShowButton,
   className,
 }: DoctorCardProps): JSX.Element {
+  const history = useHistory();
+  function navigateToProfile() {
+    history.push(`/doctor/${id}`);
+  }
```

Comment:
If this is only used to navigate to another page please use a `<Link/>`
https://reactrouter.com/web/api/Link

Reply:
Thank you for this advice!